src/flexalloc_xnvme_env.c

```diff
@@ -189,102 +173,195 @@ fla_xne_sync_seq_w_naddrs(struct xnvme_dev * dev, const uint64_t slba, const uin
   return err;
 }
 
-// Read or write out a larger chunk striped across multiple objects
-int
-fla_xne_sync_strp_seq_x(struct xnvme_dev *dev, const uint64_t offset, uint64_t nbytes,
-                        void const *buf, struct fla_sync_strp_params *sp, bool write)
+struct fla_async_strp_cb_args
```

Comment:
There is a lot of info that is reused for all of the cb_args, nsid, buf, dev_lba_nbytes are examples. I would separate out the variables that can be shared across all outstanding cbs from the info needed for an individual cb.